### DISCUSSION TITLE
How do i find advanced HaxeFlixel tutorials, if there are any?

### DISCUSSION THREAD

**Discussion by impostor5875** [HaxeFlixel/flixel#2558] [Q&A] [answered]
so basically, i did that dungeon crawler tutorial thing on the official haxeflixel website, and now i understand the basics of it.

i want to learn more, but when i try to search up tutorials on yt, all that comes up is just beginner tutorials and that's it. anyone have any tips or better tutorials?

**Comment by Geokureli** [comment, marked as answer]
We are looking to expand the HaxeFlixel tutorials section, as some people clearly prefer to learn that way. Creating tutorials is not something I'm experienced with so it may take a while. Until then I recommend picking a small project and using the existing resources to try and get there. If there are specific tutorials you would like to see, feel free to list them here.

https://snippets.haxeflixel.com/ is an ever expanding list of features with minified examples.
https://haxeflixel.com/demos/ are tiny projects that showcase features, nearly all of flixel's out-of-box capabilities are represented here, as well as some common gamedev techniques.
Richard Oliver Bray has a fantastic youtube series of HaxeFlixel tutorials
https://www.youtube.com/playlist?list=PLiKs97d-BatFGPrkf7yNN0e6IyToRaaYO

Lastly, I recommend joining [the Haxe discord](https://discordapp.com/invite/rqEBAgF). There are many experienced HF devs there that can help you make games in Flixel.

Edit: I haven't had a chance to check these out myself, but these might be of help
https://github.com/ezacharyk/BGP_Projects

**Reply by impostor5875** [reply]
alright, thanks!
